test/integration/model/locrequest.model.spec.ts

```diff
@@ -39,6 +39,12 @@ describe('LocRequestRepository', () => {
 
         expect(requests[0].getDescription().requesterAddress).toBe("5CXLTF2PFBE89tTYsrofGPkSfGTdmW4ciw4vAfgcKhjggRgZ");
         expect(requests[0].getDescription().ownerAddress).toBe("5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY");
+        expect(requests[0].getDescription().userIdentity).toEqual({
+            firstName: 'John',
+            lastName: 'Doe',
+            email: 'john.doe@logion.network',
+            phoneNumber: '+123456'
+        });
```

Comment:
Maybe add a test checking that `userIdentity` is undefined when no data have been set (and not, for example, an object with all fields set to "")?

Reply:
Good catch, `userIdentity` was indeed defined with blank fields.